compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1999,6 +2008,7 @@ object Parsers {
       case ABSTRACT    => Mod.Abstract()
       case FINAL       => Mod.Final()
       case IMPLICIT    => Mod.Implicit()
+      case IMPLIED     => Mod.Implied()
```

Comment:
That looks suspicions, likely to be a trap. Implied is not a modifier in the traditional sense. Better not to include it here.